src/common/vitals/cumulative-layout-shift.test.js

```diff
@@ -59,13 +70,11 @@ describe('cls', () => {
     getFreshCLSImport(metric => {
       metric.subscribe(({ value }) => {
         triggered++
-        expect(value).toEqual(1)
-        expect(triggered).toEqual(1)
-        setTimeout(() => {
-          expect(triggered).toEqual(1)
-          done()
-        }, 1000)
+        expect(value).toEqual(0.123)
       })
+      metric.update({ value: 0.123 })
+      expect(triggered).toBeGreaterThanOrEqual(2)
```

Comment:
I'm still curious why this behavior and test are changing to allow CLS to report identical values more than once

Reply:
The test description says reports more than once, not necessarily "except if the value is the same", so I'm just updating it to be more true to description.

Also, `onCLS` inherently will not report the same value more than once, so that behavior is baked in already

Reply:
We can keep that redundant check though. I'm just following the philosophy of letting CWV handle calculations and scoring.

Reply:
Then IMO wouldn't test that it **can** report the same value twice, since i think that implies that it has some utility for us (where in fact the test used to ensure that it can not report the same value twice since we dont want to harvest identical nodes twice)

Reply:
You suggesting to remove it? It does serve to ensure there's no check blocking it from reporting twice.

Reply:
I cant see any case where we would **want** it to report the same value twice, so we need to pick a behavior here.  Either we enforce it like it was before or not, which is fine, but testing that it **can** do something we don't want it to do is strange to me here.

Reply:
I don't normally advocate for writing tests for a third-party but, if we are assuming that the CWV library will not double-report the same value, it makes sense to me that we test that assumption just in case the library is updated later and breaks that assumption.

Reply:
(or just go back to enforcing it ourselves and not need to worry)

Reply:
The test itself is really for cls vital metrics class and the callback as onCLS is mocked, not to be conflated with the behavior of the original onCLS per se. Even though it's not expected to double report the same value in the field, unit testing that it by _our_ callback definition is allowed isn't really counterintuitive. That said, since it raised confusion, I removed it.